### DISCUSSION TITLE
Using a custom theme on Keycloak in dev mode

### DISCUSSION THREAD

**Discussion by guillaume-gorret** [quarkusio/quarkus#31821] [Q&A] [answered]
Hello !

I'm trying to use a custom theme in keycloak in dev mode and I was asking my self about an easy solution to launch the dev services with the custom theme. 

Is there any solution? Did someone already try this?

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment, marked as answer]
@IceDevelop74 Hi, looks like it should be asked in Keycloak discussions, as in Quarkus itself it cant be controlled
